### ISSUE TITLE
Filezilla hashes extraction is not working

### ISSUE THREAD

**Issue by claudioandre-br** [openwall/john#3970]
@luizcarlosrodrigues, can you share your file? At least, create one we can use to test?

Thanks

****
@kholia or @claudioandre-br  after extract filezilla hashes to a hash file and run john I got:
hash: dbug_portal:$dynamic_82$e6927a3d4eef9297080795a022dead82de0cfcbe7abfdd7975f44bafe06c2ae606fcbcdbd8b44e3ca35c60d6a4f69b6f17e24074cb301ed09baee67c41834efc$HEX$512624655a4572253e22736d7c744b66577e573821683c365b29382a6d314a5d7d263c516c4f246b3c4f22445875276065286471362633783b6e236a24594859

john hash.txt
No password hashes loaded

_Originally posted by @luizcarlosrodrigues in https://github.com/magnumripper/JohnTheRipper/pull/2374#issuecomment-493696497_

**Comment by luizcarlosrodrigues** [comment]
sure, file from my test server. Is the filezilla server xml
[filezilla.txt](https://github.com/magnumripper/JohnTheRipper/files/3194734/filezilla.txt)

**Comment by claudioandre-br** [comment]
This is working for me:

```
$ run/filezilla2john.py ~/Downloads/filezilla.txt > ~/hash

$ john ~/hash --format=dynamic_82
Using default input encoding: UTF-8
Loaded 10 password hashes with 10 different salts (dynamic_82 [sha512($p.$s) 128/128 SSE2 2x])
Warning: no OpenMP support for this hash type, consider --fork=2
Proceeding with single, rules:Single
Press 'q' or Ctrl-C to abort, almost any other key for status
```

Do you know any password? Just to check the process is really working.

**Comment by luizcarlosrodrigues** [comment]
it's a old test server, i forgot the passwords that's why i'm trying to crack.
i will try to clean everything and clone the repo again

**Comment by luizcarlosrodrigues** [comment]
one of the passwords should be a3b6c9d12

**Comment by claudioandre-br** [comment]
If the password is correct, it is NOT working

```
$ cat ~/wordlist 
a3b6c9d12
```

```
$ john ~/hash --format=dynamic_82 --word=/home/claudio/wordlist
Using default input encoding: UTF-8
Loaded 10 password hashes with 10 different salts (dynamic_82 [sha512($p.$s) 128/128 SSE2 2x])
Warning: no OpenMP support for this hash type, consider --fork=2
Press 'q' or Ctrl-C to abort, almost any other key for status
Warning: Only 1 candidate left, minimum 24 needed for performance.
0g 0:00:00:00 DONE (2019-05-19 00:23) 0g/s 5.000p/s 50.00c/s 50.00C/s a3b6c9d12
Session completed

```

**Comment by luizcarlosrodrigues** [comment]
i'm not quite sure about the password.

**Comment by claudioandre-br** [comment]
Go ahead and try to crack it. 

Notice you have to add the `--format=dynamic_82` to your command line.

**Comment by luizcarlosrodrigues** [comment]
tks for your help

**Comment by solardiz** [comment]
> Notice you have to add the `--format=dynamic_82` to your command line.

Looks like something we should fix, given that this information is available from the hash encoding in the input file.

**Comment by luizcarlosrodrigues** [comment]
any idea what type this hash is?
1000:NNbgGA3CEOmt7f19qtx6tQE81IAYHTYr:QpaNE56p88vsXqwHVe8wZk5W0CIJh6aF

**Comment by claudioandre-br** [comment]
> any idea what type this hash is?
> 1000:NNbgGA3CEOmt7f19qtx6tQE81IAYHTYr:QpaNE56p88vsXqwHVe8wZk5W0CIJh6aF

These kind of question you should ask in john-users mailing list (for real users read your message).

**Comment by luizcarlosrodrigues** [comment]
seems like john didnt work with the dynamic_82 and the file sent. just run with rockyou.txt and nothing found

**Comment by jfoug** [comment]
This is NOT working by design.  If you want dynamic_82 to run without the -format=dynamic_82, then comment out the line for dynamic_82 in the ./run/dynamic_disabled.conf

That file was added (not by me) to keep many dynamic formats from auto working.

**Comment by solardiz** [comment]
Assumed fixed by #4324
